### PR TITLE
git-status: add verbose command

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Managor** [comment]
I use this often to double check my stuff before I commit.

File: pages/common/git-status.md [hunk 1]
```diff
@@ -12,6 +12,10 @@
 
 `git status --short`
 
+- Show [v]erbose information on changes that have not yet been committed:
+
+`git status --verbose --verbose`
```
[the remark this answers is not in the document]
Comment (Managor) [reply pages/common/git-status.md:17]: It's intentional. One verbose shows staging area, two verboses shows staging and working directory.

[the remark this answers is not in the document]
Comment (Managor) [reply pages/common/git-status.md:17]: That's what I attempted to do with as few letters as possible. Changes that have not yet been commited involves both the staging area and working directory.

[the remark this answers is not in the document]
Comment (Managor) [reply pages/common/git-status.md:17]: That's probably better because it's explicit

**Comment by Managor** [comment]
Putting double verbose here instead of single verbose (which is what I normally use) is good because it sort of documents both. The user will question in their mind what happens with just one instance

**Review by gutjuri — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**